src/components/GiftCard.vue

```diff
@@ -27,21 +27,18 @@
 </template>
 <script>
 import genQrCode from './GenQRCode'
+import {date} from 'quasar'
 export default {
   props: {
-    giftCard: {
-      type: Object,
-      default: () => {},
-    },
+    giftCard: [Object],
```

Comment:
this component is obviously a GiftCard, so the props for the GiftCard should be:
code, amount, **createdAt** and expiry which all have a default value
Dont create a GiftCard component with a single GiftCard prop